hack/swift/Makefile

```diff
@@ -94,11 +96,11 @@ overlay-byocni-up: rg-up overlay-net-up ## Brings up an Overlay BYO CNI cluster
 
 overlay-cilium-up: rg-up overlay-net-up ## Brings up an Overlay AzCNI cluster
 	$(AZCLI) aks create -n $(CLUSTER) -g $(GROUP) -l $(REGION) \
-		--node-count 2 \
+		--node-count $(NODE_COUNT) \
 		--node-vm-size $(VM_SIZE) \
 		--load-balancer-sku basic \
 		--network-plugin azure \
-		--enable-cilium-dataplane \
+		--network-dataplane cilium \
```

Comment:
did this work? i think azcli support not released for --network-dataplane

Reply:
Yes it works.

Reply:
can you update the other up targets with these changes also? keep them all consistent

Reply:
Sure